<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPage</class>
 <widget class="QWidget" name="SearchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>644</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameSearch">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <item>
       <widget class="QLabel" name="labelSearchFor">
        <property name="text">
         <string>&amp;Search for:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSearch</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Examples:&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;DuplicateHandle&lt;/span&gt; -&amp;gt; searches for the word &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;DuplicateHandle&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;DuplicateHandle*&lt;/span&gt; -&amp;gt; search for all words which start with &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;DuplicateHandle&lt;/span&gt;. The asterix can also be used in front or in the middle of words.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;GetWindowText(hWnd*,&lt;/span&gt; -&amp;gt; searches for &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;GetWindowText&lt;/span&gt; followed by &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;(&lt;/span&gt; followed by all words which start with &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;hWnd &lt;/span&gt;and finally followed by &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;a* * b&lt;/span&gt; -&amp;gt; searches for all words starting with &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;a&lt;/span&gt; followed by the character &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;*&lt;/span&gt; and the character &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;b&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string>Find</string>
        </property>
        <property name="icon">
         <iconset resource="SearchPage.qrc">
          <normaloff>:/default/resources/Find.png</normaloff>:/default/resources/Find.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCustomScripts">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Search the current location using a custom search script&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="SearchPage.qrc">
          <normaloff>:/default/resources/CustomSearch.png</normaloff>:/default/resources/CustomSearch.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLockResultSet">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Search in the current search results. After clicking this button all further searches will return only matches from the current search results.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="SearchPage.qrc">
          <normaloff>:/default/resources/Lock.png</normaloff>:/default/resources/Lock.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFolderFilter">
        <property name="text">
         <string>Filter by file/&amp;folder:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboFolderFilter</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboFolderFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A comma seperated list of strings which must appear in the file path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelExtensionFilter">
        <property name="text">
         <string>Filter by &amp;extension:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboExtensionFilter</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboExtensionFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A comma seperated list of extensions. You can type &lt;span style=&quot; font-weight:600;&quot;&gt;.ext&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;ext&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;*.ext&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCaseSensitive">
        <property name="text">
         <string>Case sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelLocation">
        <property name="text">
         <string>&amp;Location:</string>
        </property>
        <property name="buddy">
         <cstring>comboLocation</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboLocation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frame">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelMatches">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>0 matches</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonExport">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="SearchPage.qrc">
                <normaloff>:/default/resources/Export.png</normaloff>:/default/resources/Export.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonInfo">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="SearchPage.qrc">
                <normaloff>:/default/resources/PerfInfo.png</normaloff>:/default/resources/PerfInfo.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="PathDragListView" name="listView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>165</red>
                  <green>212</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>165</red>
                  <green>212</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>10</red>
                  <green>36</green>
                  <blue>106</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="SourceViewer" name="sourceViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourceViewer</class>
   <extends>QWidget</extends>
   <header>SourceViewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathDragListView</class>
   <extends>QListView</extends>
   <header>PathDragListView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SearchPage.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>SearchPage</receiver>
   <slot>fileSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>SearchPage</receiver>
   <slot>performSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>SearchPage</receiver>
   <slot>openFileWithSystem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>SearchPage</receiver>
   <slot>contextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonInfo</sender>
   <signal>clicked()</signal>
   <receiver>SearchPage</receiver>
   <slot>performanceInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>838</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonExport</sender>
   <signal>clicked()</signal>
   <receiver>SearchPage</receiver>
   <slot>exportMatches()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCustomScripts</sender>
   <signal>clicked()</signal>
   <receiver>SearchPage</receiver>
   <slot>execCustomScripts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SearchPage</receiver>
   <slot>fileSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboLocation</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>SearchPage</receiver>
   <slot>currentLocationChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1111</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonLockResultSet</sender>
   <signal>clicked(bool)</signal>
   <receiver>SearchPage</receiver>
   <slot>lockResultSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileSelected(QModelIndex)</slot>
  <slot>performSearch()</slot>
  <slot>openFileWithSystem()</slot>
  <slot>contextMenuRequested(QPoint)</slot>
  <slot>performanceInfo()</slot>
  <slot>exportMatches()</slot>
  <slot>execCustomScripts()</slot>
  <slot>currentLocationChanged()</slot>
  <slot>lockResultSet(bool)</slot>
 </slots>
</ui>
